sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -777,8 +785,27 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 			if (!links[id].cpus->dai_name)
 				goto devm_err;
 		}
+		id++;
 	}
 
+	/* CNVi bt-offload */
```

Comment:
Not sure if CNVi means anything for upstream?

Reply:
Ack, will remove the Intel "jargon".